api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -109,5 +116,65 @@ public void getOrderGroupsByPatient_shouldGetOrderGroupsGivenPatient() {
 		Patient existingPatient = Context.getPatientService().getPatient(8);
 		List<OrderGroup> ordergroups = Context.getOrderService().getOrderGroupsByPatient(existingPatient);
 		assertEquals(1, ordergroups.size());
+		
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderGroupAttributeTypes()}
+	 * @throws Exception
+	 */
+    @Test
+	public void getAllOrderGroupAttributeTypes_shouldGetAllOrderGroupAttributeTypes(){
+		List<OrderGroupAttributeType> orderGroupAttributeTypes = dao.getAllOrderGroupAttributeTypes();
+		assertEquals(orderGroupAttributeTypes.size(),4);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupAttributeTypeByUuid(String)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderGroupAttributeType_shouldGetOrderGroupAttributeTypeGivenUuid(){
+		final String UUID2 ="9cf1bdb2-d18e-11ea-87d0-0242ac130003";
+    	OrderGroupAttributeType newOrderGroupAttributeType = dao.getOrderGroupAttributeTypeByUuid(UUID2);
+		assertEquals(newOrderGroupAttributeType.getName(),dao.getOrderGroupAttributeTypeByUuid(UUID2).getName());
```

Comment:
What exactly are you testing here? Do you take a look at existing tests to see how we do it?